lib/commands/crafter/add_office.rb

```diff
@@ -25,21 +30,25 @@ def updated_apprentice
     end
 
     def run
-      if Office.available?(@office) && (@crafter || @apprentice)
+      if @office.available? && (@crafter || @apprentice)
```

Comment:
We could remove this condition as we are doing the verification after

```suggestion
      if @office.available?
```

or have
```
if @office.available? && @crafter
...
end
```

```
if @office.available? && @apprentice
...
end
```